unconstrained_test.go

```diff
@@ -776,11 +1189,11 @@ func testLocal(t *testing.T, tests []unconstrainedTest, method Method) {
 			continue
 		}
 
-		f := newFunctionInfo(test.f)
+		p := newProblemInfo(test.p)
```

Comment:
I think this is not necessary anymore, it existed to avoid the type assertions. You can use `test.p` directly below.